<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="champ_clear_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>21</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>비우기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_champ_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="champ_list_widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>361</width>
     <height>231</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="champ_ok_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
